packages/ui-kit/src/components/markdown.js

```diff
@@ -245,6 +245,10 @@ const TypographyPage = styled.div`
     margin-top: ${dimensions.spacings.m};
   }
 
+  section:first-of-type > * {
```

Comment:
I think the selector is matching too much.  The requirement is to only match the very first content element on a given page, not any direct descendant of the first section. 

```suggestion
  section:first-of-type > *:first-child {
```